php/src/vaas/Vaas.php

```diff
@@ -190,15 +183,12 @@ public function ForFile(string $path, $upload = true, string $uuid = null): Vaas
      * @throws GuzzleException
      * @throws UploadFailedException
      */
-    public function ForStream(Stream $stream, string $uuid = null): VaasVerdict
+    public function ForStream(ReadableStream $stream, int $size = 0, string $uuid = null): VaasVerdict
     {
-        if ($uuid == null) {
-            $uuid = UuidV4::getFactory()->uuid4()->toString();
-        }
-
-        $verdictResponse = $this->_verdictResponseForStream(
-            $uuid
-        );
+        $this->_logger->debug("uuid: ".var_export($uuid, true));
```

Comment:
Considering that uuid is typed to be a string, isn't the var_export a bit too much here? A simple . $uuid should suffice I think.